client/src/client.ts

```diff
@@ -136,7 +136,26 @@ function registerNotificationHandlers(
         }
       });
 
-  context.subscriptions.push(disposable1, disposable2);
+  const disposable3 = client.onNotification(
+      SuggestIvyLanguageService, async (params: SuggestIvyLanguageServiceParams) => {
+        const config = vscode.workspace.getConfiguration();
+        if (config.get('angular.enable-experimental-ivy-prompt') === false) {
+          return;
+        }
+
+        const enableIvy = 'Enable';
+        const selection = await vscode.window.showInformationMessage(
+            params.message,
+            enableIvy,
+            'Ignore',
+        );
+        if (selection === enableIvy) {
+          config.update('angular.experimental-ivy', true);
```

Comment:
Does the language service automatically restart once the config is updated?

Reply:
Yep, it looks like it restarts whenever any configuration changes.